src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -170,7 +170,8 @@ def load_arguments(self, _):
         c.argument('slot', options_list=['--slot', '-s'], help='Name of the web app slot. Default to the productions slot if not specified.')
 
     with self.argument_context('webapp list-runtimes') as c:
-        c.argument('linux', action='store_true', help='list runtime stacks for linux based web apps')
+        c.argument('linux', options_list=['--linux', '-l'], action='store_true', help='list runtime stacks for linux based web apps')
+        c.argument('windows', options_list=['--windows', '-w'], action='store_true', help='list runtime stacks for windows based web apps')
```

Comment:
@seligj95  & @StrawnSC was this the recommendation from the spec. Instead of adding --windows , can we add --os which will take linux or windows as values & mark --linux as something that to be deprecated in the future . This will ensure simplicity in the usage of command & matches other commands like create & up where we use --os.
Marking the --linux as to be deprecated in future will ensure this is not a total breaking change wrt command signature & we can eventually remove this. Thoughts?

Reply:
yeah I think it would make sense to have similar usage to `az webapp up`, which uses an `--os-type` parameter. So that argument would be not required and the default behavior would be to show linux and windows stacks, right?

Reply:
Correct --os would be optional, exemption of this, default should be both linux & windows. Thanks!

Reply:
agreed with Sisira's thoughts. we should deprecate --linux following the standard process